charts/drax/charts/service-orchestrator/templates/configmap.yaml

```
@@ -10,6 +10,13 @@ top:
   {{ $ | toYaml | nindent 2 }}
 
 name: {{ include "accelleran.common.fullname" (dict "top" $) }}-config
+{{- $config := .Values.config }}
+{{- $instance := include "accelleran.common.instance" (dict "top" .) }}
+
+{{- if and (hasKey $config "serviceMonitorUrl") (eq ($config.serviceMonitorUrl | default "") "") }}
```

Instead of the `eq (<something> | default "") ""` it would be better to use `empty <something>`.

Also the key doesn't need to be checked again as a field access in combination with braces around the object to access the field in (not needed in this case as we have the variable) results in nil on it's own (which is `empty`).

```suggestion
{{- if empty $config.serviceMonitorUrl }}
```